bundles/org.openhab.binding.venstarthermostat/pom.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.openhab.addons.bundles</groupId>
+    <artifactId>org.openhab.addons.reactor.bundles</artifactId>
+    <version>2.5.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>org.openhab.binding.venstarthermostat</artifactId>
+
+  <name>openHAB Add-ons :: Bundles :: Venstar Thermostat Binding</name>
+
+  <dependencies>
+    <!-- https://mvnrepository.com/artifact/org.eclipse.jetty/jetty-client -->
+    <dependency>
+      <groupId>org.eclipse.jetty</groupId>
+      <artifactId>jetty-client</artifactId>
```

Comment:
This library is already available via core and should not be added here. Any reason to add it here?

Reply:
There's a dependency on jetty-client 9.4 because there's a bug in 9.3 (related to its use in discovery, IIRC).

Reply:
The version used during compile of openHAB is 9.3, but the runtime version is: 9.4.20.v20190813 see: https://github.com/openhab/openhab-core/blob/master/bom/runtime/pom.xml
So you don't needs this here.